app/models/person.rb

```diff
@@ -247,7 +224,7 @@ def update_remote_attributes(remote_attributes)
   private
 
   def update_nation_builder
-    relevant_fields = changed & FIELDS_ALSO_ON_NB
+    relevant_fields = changed & %w[email first_name last_name is_volunteer phone]
```

Comment:
this method is going to be deleted as soon as I rewrite the method I asked you about in Slack. I moved these fields here temporarily so I could delete `FIELDS_ALSO_ON_NB` from the top of this class